src/cache.ts

```diff
@@ -0,0 +1,45 @@
+import type { Evaluation } from './types'
+
+export function getCacheId(targetIdentifier: string): string {
+  return 'HARNESS_FF_CACHE_' + targetIdentifier
+}
+
+export function loadFromCache(targetIdentifier: string): Evaluation[] {
+  const cachedEvaluations = window.localStorage.getItem(getCacheId(targetIdentifier))
+
+  if (cachedEvaluations) {
+    try {
+      return JSON.parse(cachedEvaluations)
+    } catch (e) {}
```

Comment:
what should it do with any error it catches?

Reply:
It should shut up and keep going

Reply:
This will only happen if the value is not valid JSON, so we can assume there are no evals to restore

Reply:
^ what I tell myself everyday

Reply:
Words to live by!